src/frontend/src/btc/utils/btc-transactions.utils.ts

```diff
@@ -0,0 +1,59 @@
+import type { BtcTransactionStatus, BtcTransactionUi } from '$btc/types/btc';
+import type { BtcAddress } from '$lib/types/address';
+import type { BitcoinOutput, BitcoinTransaction } from '$lib/types/blockchain';
+import type { TransactionType } from '$lib/types/transaction';
+import { nonNullish } from '@dfinity/utils';
+
+export const mapBtcTransaction = ({
+	transaction,
+	btcAddress
+}: {
+	transaction: BitcoinTransaction;
+	btcAddress: BtcAddress;
+}): BtcTransactionUi => {
+	const type: TransactionType = transaction.inputs.some(
```

Comment:
nitpick: Our code style leverages destructing objects. If you could follow it, it would be nice

Reply:
Sure, done.